pages/archive/content.py

```diff
@@ -85,5 +135,20 @@ def stable_content(self):
 
     @property
     def stable_dict(self):
-        return {**{field: self.dict[field] for field in self._stable_fields},
-                **{'content': self.stable_content}}
+        """Returns a stable version of the full archive json as a dict.
+
+        Returns from the archive json only fields that are
+        guaranteed not to change from test to test as a dict.
+
+        This includes the whitelisted fields in the _stable_fields array,
+        plus stable versions of the tree and content fields, if present.
+        """
+        dict = {field: self.dict[field] for field in self._stable_fields if field in self.dict}
```

Comment:
dict is a built-in keyword in python. you may want to choose a different variable name.